lib/node_modules/@stdlib/math/base/special/besselj0/README.md

```diff
@@ -0,0 +1,68 @@
+# besselj0
+
+> Compute the [Bessel function of the first kind][bessel] of order zero.
+
+
+<section class="usage">
+
+## Usage
+
+``` javascript
+var besselj0 = require( '@stdlib/math/base/special/besselj0' );
+```
+
+#### besselj0( x )
+
+Compute the [Bessel function of the first kind][bessel] of order zero at `x`.
+
+``` javascript
+var v = besselj0( 0.0 );
+// returns 1.0
+
+v = besselj0( 1.0 );
+// returns 0.76519768655796649
+
+v = besselj0( Infinity );
```

Comment:
Do you prefer importing PINF/NINF and using that in these examples, or is that unnecessarily verbose?

Reply:
Import.